doc/_static/css/custom_styles.css

```
@@ -21,6 +21,11 @@ strong {
 
 /* Style for Font Awesome that only styles the icons before elements */
 
+.openteams-icon::before {
+  content: url("../images/Opeteams_Favicon-01.svg");
+  width: 1em;
+}
+
```

```suggestion

/* Style for OpenTeams icon, used like the FA Brands icons */

.openteams-icon::before {
  content: url("../images/openteams-favicon.svg");
  width: 1.3em;
  margin-left: -0.2em;
}

div.openteams-icon::before {
  margin-right: 0;
  padding-bottom: 0;
}

```

This fixes the faux-FAB icon's size and alignment not matching the others, fixes the file name and adds an appropriate descriptive comment. Aside from these changes, you'll need to move this to the bottom of the FA icons section (just above the first `iframe` rule), as it is a bespoke override to them (so the styles apply correctly, as well as it makes more sense).